python/phylanx/ast/physl.py

```diff
@@ -383,12 +383,17 @@ def _compile_or_load(self):
                 check_return(self.ir)
 
                 self.__src__ = self._generate_physl(self.ir)
+                if self.doc_src is not None:
+                    self.__src__ = self.doc_src
                 self.__ast__ = phylanx.ast.generate_ast(self.__src__)
 
                 # now store the PhySL string and AST for this function
                 physl_db.insert(
                     self.wrapped_function.__name__, self.__src__, self.__ast__)
 
+            if self.doc_src is not None:
+                self.__src__ = self.doc_src
```

Comment:
Why is replace `__src__` necessary here? It was either successfully loaded from the database of set above.